scripts/mkcloud

```diff
@@ -407,18 +407,23 @@ function setupadmin
     onadmin write_cloud_info
     echo "you can now proceed with installing crowbar"
     # prevent jumbo frames from going out
+    local IPTABLES_WAITOPT=
+    if $($sudo iptables --wait --list INPUT > /dev/null); then
```

Comment:
`sudo iptables --help | grep -P -- --wait\\t` can be more explicit, maybe even removing the `tab`